### PR TITLE
Allow user to interrupt launch after 3 tries

### PR DESCRIPTION
**Pull request by Scrumplex**
Also remove unused variables

### CODE REVIEW

File: launcher/LaunchController.cpp [hunk 1]
```diff
@@ -145,16 +145,25 @@ void LaunchController::login() {
...
+        if (tries > 0 && tries % 3 == 0) {
+            auto result = QMessageBox::question(
+                m_parentWidget,
+                tr("Continue launch?"),
+                tr("It looks like we couldn't launch after %1 tries. Do you want to continue trying?")
+                    .arg(tries)
+            );
+
+            if (result == QMessageBox::No) {
+                return;
```
Comment (flowln) [inline launcher/LaunchController.cpp]:
```suggestion
            if (result == QMessageBox::No) {
                emitAborted();
                return;
```
github's diff is weird o.O

**Comment by DioEgizio** [comment]
get dcod

**Comment by txtsd** [comment]
How does one test this?

**Comment by Scrumplex** [comment]
> How does one test this?

I commented out all of the following code, which will make the launcher think that your accounts needs a refresh:
https://github.com/Scrumplex/PolyMC/blob/fix-infinite-auth-loop/launcher/LaunchController.cpp#L184-L249

**Review by txtsd — APPROVED** [review]
LGTM!

**Review by flowln — APPROVED** [review]

**Review by timoreo22 — APPROVED** [review]

**Pull request merged**